styles.css

```diff
@@ -1,5 +1,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&family=Roboto:wght@400;700&display=swap');
-
+@import url("/Component/Products/product.css");
 
 *{
```

Comment:
Generally colors are defined in root selector

Reply:
i did defined in root only. can you check again

Reply:
Oh okay. Sorry, I got confused